project/Dependencies.scala

```diff
@@ -74,7 +74,7 @@ object Dependency {
     val Chaos = "0.8.7"
     val Guava = "19.0"
     // FIXME (gkleiman): reenable deprecation checks after Mesos 1.0.0-rc2 deprecations are handled
```

Comment:
I think so. We were referencing some deprecated proto fields for volumes until they were officially removed from the mesos API. I don't think we got to that point yet.